.gitignore

```diff
@@ -0,0 +1 @@
+/.idea/
```

Comment:
This is just to tell git to ignore a folder used by my editor. It also shows you what a gitignore file looks like.